src/components/Homepage/LendByCategory/LoanCategoriesSection.vue

```diff
@@ -12,7 +12,7 @@
 				:class="{'active': category.id === activeCategory}"
 				v-for="category in prefetchedCategoryInfo"
 				:key="category.id + '-link'"
-				@click.prevent="setActiveCategory(category.id)"
+				@click.prevent.native="setActiveCategory(category.id)"
```

Comment:
I've forgotten this small detail several times now. KvButton needs the .native decorator on the click event to work when it is triggering a component function.

Reply:
It'd be nice to not need that at all and just use @click. Pretty sure we can make changes in KvButton to facilitate that. Will make a ticket in the Vue backlog.